plugins/forc-index/src/ops/forc_index_start.rs

```diff
@@ -120,7 +120,7 @@ pub async fn init(command: StartCommand) -> anyhow::Result<()> {
         }
     }
 
-    println!("{cmd:?}");
+    info!("{cmd:?}");
```

Comment:
So for this, and _all_ `Command`s, could we only `info!` log them if `-v --verbose` is set?

Reply:
Sure

Reply:
by the looks of it we only `info!("{cmd:?}")` in `forc_index_start` and in no other ops file.